src/native/fs/gpfs_fcntl.h

```diff
@@ -456,7 +533,7 @@ typedef struct {
 
 typedef struct BadBlockInfo {
   long long nBadBlocks;         /* number of bad blocks */
-  BadBlock badBlocks[1];         /* bad block info array */
+  BadBlock badBlocks[0];         /* bad block info array */
```

Comment:
@jackyalbo FYI - you originally added this header file and manually changed arrays from 0 to 1 because our `-pedantic` compilation does not allow zero size arrays, I changed now to use the header file as is and fixed the compilation to allow it (using `#pragma GCC diagnostic ignored "-Wpedantic"` around the gpfs includes in fs_napi).